File: java/src/main/java/br/com/acqio/dto/InstallmentInfo.java
package br.com.acqio.dto;

import br.com.acqio.enums.CardApplication;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.math.BigDecimal;

@Getter
@Setter
@ToString
public class InstallmentInfo {
    private String cardBrand;
    private CardApplication cardApplication;
    private Integer installment;
    private BigDecimal installmentValue;
    private BigDecimal total;
}

File: csharp/csharp/dto/InstallmentInfo.cs
namespace csharp.dto
{
    class InstallmentInfo
    {
        public string cardBrand { get; set; }
        public CardApplication cardApplication { get; set; }
        public int installment { get; set; }
        public decimal installmentValue { get; set; }
        public decimal total { get; set; }

    }
}